tools/Environments/DevHome.Environments/Strings/en-us/Resources.resw

```diff
@@ -358,5 +358,8 @@
   <data name="MoreOptionsButton.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
     <value>More options</value>
     <comment>Name of the button that brings up the "More options" menu</comment>
+  <data name="Operations_ApplyConfiguration" xml:space="preserve">
+    <value>Setup</value>

```

Comment:
"Set up" as a verb is two words. Or do we mean the noun? (We get this wrong so often and @cinnamon-msft always catches it 🙂)